scripts/portconfig

```diff
@@ -60,7 +60,7 @@ SWITCH_CAPABILITY = "SWITCH_CAPABILITY|switch"
 # STATE_DB constants
 PORT_STATE_TABLE_NAME = "PORT_TABLE"
 PORT_STATE_SUPPORTED_SPEEDS = "supported_speeds"
-
+PORT_STATE_SUPPORTED_FECS = "supported_fecs"
```

Comment:
It depends on https://github.com/Azure/sonic-swss/pull/2333.
You can find more information in the PR description:
The supported FEC mode is exposed to `STATE_DB.PORT_TABLE|<port>.supported_fecs` by orchagent
CLI checks whether the FEC mode to be set is in the `supported_fecs` list before configuring it to CONFIG_DB.
Specifically, in case the vendor SAI doesn’t support fetching supported FEC mode, `N/A` should be exposed. In this case, any FEC model should be supported.